### DISCUSSION TITLE
Can't use gradle package when --offline

### DISCUSSION THREAD

**Discussion by wheelergames** [community/community#37931] [Packages]
I am attempting to host my own library on github packages, and then to use it `--offline` when using gradle on the command line later on.

For some reason I get the error 

```No cached version of [MY LIBRARY] available for offline mode```

When working online, I see the library is downloaded and saved in my `~/.gradle/caches` directory, so it's definitely there.

Also, it's only packages hosted on github packages that are failing to be found, all other libraries work from the `~/.gradle/caches` directory just fine.

Has anyone had experience with this?

**Comment by ossanna16** [comment]
Hi there @wheelergames and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
